src/style.css

```diff
@@ -0,0 +1,52 @@
+body {
+  margin: 0;
+}
+
+.stars__star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  margin-left: 4px;
+  background-position: 50%;
+  /* background-image: url("./images/star-active.svg"); */
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star:first-child {
+  margin-left: 0;
+}
+
+/* .stars--0:nth-child(n),
+.stars--1:nth-child(n + 2),
+.stars--2:nth-child(n + 3),
+.stars--3:nth-child(n + 4),
+.stars--4:nth-child(n + 5) {
+  background-image: url("./images/star.svg");
+} */
+
```

Comment:
why did you comment loop? pls use loop for styles instead of adding styleone by one